### DISCUSSION TITLE
Issues on Samsung A52

### DISCUSSION THREAD

**Discussion by Ncit** [sceneview/sceneview#163] [Q&A]
I have issues with model on devices samsung a52 and simulator. Can you give me some advices, on how to fix it?
On test device honor 10 everything is ok.
These is model on samsung a52
![image_2022_09_23T13_18_54_198Z](https://user-images.githubusercontent.com/2179841/192256580-c1404469-00e3-474b-80a9-2ef301554d32.png)
This is model in simulator
![image](https://user-images.githubusercontent.com/2179841/192257306-02e16862-9639-4ad7-8635-cf7501c79494.png)

**Comment by ThomasGorisse** [comment]
Device-specific issues on older Samsung devices are often related to GPU driver bugs in the Adreno/Mali GPUs. v4.0.0-rc.1 includes Filament 1.70.2 which has improved driver compatibility.

If you're still seeing issues, please try upgrading to v4.0.0-rc.1 and report back with the specific error/behavior. Device-specific rendering bugs are usually Filament/GPU-level issues, and newer Filament versions have better workarounds for known driver bugs.
